src/ScoreDrivenModels.jl

```diff
@@ -36,11 +36,12 @@ include("distributions/chi.jl")
 include("distributions/chisq.jl")
 include("distributions/exponential.jl")
 include("distributions/gamma.jl")
+include("distributions/locationscalestd.jl")
```

Comment:
I would change std on both occasions to tdist to avoid confusions with `std`

Reply:
I changed to tdist and changed the name order of location scale